packages/oneko.rb

```diff
@@ -0,0 +1,49 @@
+require 'package'
+
+class Oneko < Package
+  description 'A cat chases around your mouse cursor.'
+  homepage 'http://www.daidouji.com/oneko/'
+  version '1.2.sakura.6'
+  compatibility 'all'
+  source_url 'https://httpredir.debian.org/debian/pool/main/o/oneko/oneko_1.2.sakura.6.orig.tar.gz'
+  source_sha256 'd89cee8b81cdb40ef23b3457c9a7fe1b0ff130081b21a41ec6c41cda01391d25'
+  
+  depends_on 'libx11'
+  depends_on 'libxext'
+  depends_on 'sommelier'
```

Comment:
Can we put this last?

Reply:
I guess it doesn't matter since the build dependencies won't be installed anyway.